Resources/public/templates/tabs/locations.hbt

```diff
@@ -23,6 +24,11 @@
                         {{#each locations}}
                             <tr>
                                 <td>
+                                    <input type="checkbox" value="{{ id }}" class="ez-location-checkbox"
+                                        data-location-id="{{ id }}"{{#if isMainLocation}} disabled{{/if}}
+                                        data-main-location="{{#if isMainLocation}}1{{else}}0{{/if}}">
```

Comment:
According to: http://www.w3.org/html/wg/drafts/html/master/syntax.html#void-elements (8.1.2.1 Start tags) it has no effect